web/app/scripts/navbar/navbar-controller.js

```diff
@@ -27,6 +30,7 @@
         }
 
         $rootScope.$on('$stateChangeSuccess', function(event, toState) {
+            ctl.authenticated = AuthService.isAuthenticated();
```

Comment:
I think it'd be good to perhaps use this variable in the navbar template with an ng-if to not render the dropdowns until the user is authenticated. Otherwise there will be failed xhr requests for those resources and the dropdowns will be empty.